### ISSUE TITLE
Admin can expire password of LDAP user

### ISSUE THREAD

**Issue by phil-davis** [owncloud/password_policy#30]
Steps to reproduce:
1. setup LDAP on your system - e.g. https://github.com/owncloud/core/wiki/How-to-test-user_ldap
2. from the command line, attempt to expire the password of an LDAP user:
```
./occ user:expire-password zombie5
The password for zombie5 is set to expire on 2018-07-09 04:01:13 UTC.
```
3. Try to login as ``zombie5``

Expected behavior:
The ``user:expire-password`` command should fail for an LDAP user.

Actual behavior:
The password gets expired (from the local point of view of ownCloud).
The user ``zombie5`` is taken to the password change page when they try to login to the webUI.
If they fill in old and new passwords..., then "Internal Server Error" is reported.

**Comment by phil-davis** [comment]
PR  to fix #31

**Comment by phil-davis** [comment]
PR merged.
